react/OrderPayment.tsx

```diff
@@ -166,6 +191,7 @@ export const OrderPaymentProvider: React.FC<OrderPaymentProps> = ({
       interestValue,
       referenceValue,
       cardLastDigits,
+      paymentsValid,
```

Comment:
The linter doesn't complain that `setCardFormFilled` is missing in the dependencies array?

Reply:
no, because the linter knows the setter function of a `useState` has a stable identity, so we can omit it from the deps array